wdio/test/configs/android.conf.js

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+const {join} = require('path');
+const {config} = require('./shared.conf');
+
+// ============
+// Specs
+// ============
+config.specs = ['./test/specs/**/*.test.js'];
+
+// ============
+// Capabilities
+// ============
+// For all capabilities please check
+// http://appium.io/docs/en/writing-running-appium/caps/#general-capabilities
+config.capabilities = [
+    {
+        automationName: 'UiAutomator2',
+        deviceName: 'Android Emulator',
+        platformName: 'Android',
+        platformVersion: '8.1',
+        orientation: 'PORTRAIT',
+        maxInstances: 1,
+        app: join(process.cwd(), '../Mattermost_Beta.apk'),
```

Comment:
this requires the app to be built first, do we need a step for that?

Reply:
@saturninoabril Any disagreement to have this be a separate ticket to handle the building?

Reply:
For reference, https://mattermost.atlassian.net/browse/MM-16301